dipy/viz/horizon/app.py

```diff
@@ -592,6 +593,15 @@ def build_show(self, scene):
 
         sync_peaks = False
         if len(self.pams) > 0:
+            for idx, f_pam in enumerate(self.pams):
+                title = f"Peaks {idx + 1}"
+                pam, fname = unpack_pams(f_pam)
```

Comment:
`unpack_pams` is useless and bring more code maintenance. 

you can do:  `pam, fname = f_pam` or use `unpack_data` which is explicit enough.